.circleci/config.yml

```diff
@@ -853,6 +948,13 @@ jobs:
 
   docs-build:
     <<: *base-job
+    parameters:
+      xcode_version:
+        type: string
+        default: '14.3'
```

Comment:
Same here, we have 14.3.0 in other places

Reply:
good call